contracts/test/helpers.js

```diff
@@ -210,10 +210,11 @@ const getOracleAddress = async (deployments) => {
  * This first sets the ETH price in USD, then token price in ETH
  *
  * @param {string} tokenSymbol: "DAI", USDC", etc...
+ * @param {string} tokenAddress: 0x....
  * @param {number} usdPrice: price of the token in USD.
  * @returns {Promise<void>}
  */
-const setOracleTokenPriceUsd = async (tokenSymbol, usdPrice) => {
+const setOracleTokenPriceUsd = async (tokenSymbol, tokenAddress, usdPrice) => {
```

Comment:
Rather than have the test user specify the address, which adds a lot of extra work and unclarity to the tests, let's just poke at the oracle ourselves here, with or without a cache, to find out what the decimals should be.